CHANGES.txt

```diff
@@ -1,7 +1,7 @@
 2.0.6 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Form autofocus on first field is now configurable.
```

Comment:
Suggest adding a reference to this PR, e.g., "See https://github.com/Pylons/deform/pull/365".